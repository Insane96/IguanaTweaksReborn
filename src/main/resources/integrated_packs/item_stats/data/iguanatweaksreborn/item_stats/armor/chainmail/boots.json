{
	"item": "minecraft:chainmail_boots",
	"durability": 30,
	"armor": 1.8,
	"modifiers": [

	]
}