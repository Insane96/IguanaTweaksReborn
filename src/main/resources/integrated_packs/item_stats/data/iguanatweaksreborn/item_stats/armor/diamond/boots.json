{
	"item": "minecraft:diamond_boots",
	"durability": 180,
	"armor": 2,
	"modifiers": [
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Armor slowdown",
			"slot": "feet",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.012,
			"operation": "multiply_base"
		}
	]
}