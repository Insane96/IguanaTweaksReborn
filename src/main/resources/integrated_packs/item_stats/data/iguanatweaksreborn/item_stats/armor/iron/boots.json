{
	"item": "minecraft:iron_boots",
	"durability": 84,
	"armor": 1.2,
	"modifiers": [
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Armor slowdown",
			"slot": "feet",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.0075,
			"operation": "multiply_base"
		},
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Item stats data pack",
			"slot": "feet",
			"attribute": "iguanatweaksreborn:innate_resistance",
			"amount": 0.4,
			"operation": "addition"
		}
	]
}