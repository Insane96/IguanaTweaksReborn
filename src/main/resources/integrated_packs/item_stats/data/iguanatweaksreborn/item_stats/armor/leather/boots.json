{
	"item": "minecraft:leather_boots",
	"durability": 48,
	"armor": 0.9,
	"modifiers": [
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Armor slowdown",
			"slot": "feet",
			"attribute": "minecraft:generic.movement_speed",
			"amount": 0,
			"operation": "multiply_base"
		}
	]
}