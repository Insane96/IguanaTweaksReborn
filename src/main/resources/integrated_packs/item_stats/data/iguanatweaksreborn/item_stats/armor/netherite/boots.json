{
	"item": "minecraft:netherite_boots",
	"durability": 168,
	"armor": 2.4,
	"modifiers": [
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Armor slowdown",
			"slot": "feet",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.018,
			"operation": "multiply_base"
		}
	]
}