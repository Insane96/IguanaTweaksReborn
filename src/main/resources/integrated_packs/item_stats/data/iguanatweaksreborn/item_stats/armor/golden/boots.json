{
	"item": "minecraft:golden_boots",
	"durability": 60,
	"armor": 1.1,
	"modifiers": [
		{
			"uuid": "dc49f564-489f-4f70-ab50-ce85cc4bfa85",
			"name": "Golden armor bonus max health",
			"slot": "feet",
			"attribute": "minecraft:generic.max_health",
			"amount": 5,
			"operation": "addition"
		},
		{
			"uuid": "4b8573c6-bd4b-4560-a411-04b78ae950ce",
			"name": "Armor slowdown",
			"slot": "feet",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.003,
			"operation": "multiply_base"
		}
	]
}